meeting_minutes/220714_SARIF_TC_62.md

```diff
@@ -0,0 +1,158 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Member                  |
+| Aharon     | Abadi     | WhiteSource       | Member                  |
+| Chris      | Meyer     | Microsoft         | Voting Member           |
+| Chris      | Wysopal   | Veracode          | Voting member           |
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Johnathan  | Gilday    | Contrast Security | Member                  |
+| Mary       | Martin    | Microsoft         | Member                  |
+| Michael    | Fanning   | Microsoft         | Voting member           |
+| Paul       | Anderson  | Grammatech, Inc.  | Voting member           |
+| Pavel      | Mikula    | Sonar             | Voting member           |
+| Stefan     | Hagen     | Individual        | Secretary, taking notes |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+| Yossi      | Banai     | Microsoft         | Member                  |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70160/agenda_20220714.html
+
+Agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-06-30_Meeting #61](https://www.oasis-open.org/committees/document.php?document_id=70117&wg_abbrev=sarif)
+
+Minutes were approved.
+
+## 1.6 Review of action items and resolutions (Co-Chair David)
+
+* ACTION on Michael to initiate contact with OpenSSF
+  * COMPLETE
+* ACTION on Chris M. to solicit a working example from WhiteSource of their scenario
+  * ONGOING
+ 
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+* Yossi Barnai will obtain voting rights at the end of next meeting if he attends.
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Johnathan Gilday will obtain voting rights at the end of this meeting.
```

Comment:
Almost right except for Jonathan.  He doesn't really fit into a category here, but we can sort of put him in section 1.7.1 because this was his first meeting since changing status from observer to member.  He will obtain voting rights at the end of next meeting if he attends, not this meeting.

Reply:
Followed ...